setup/paths.sh

```diff
@@ -37,38 +37,27 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
 
+# Export project
+export GEM_PLOTTING_PROJECT=$BUILD_HOME/gem-plotting-tools
```

Comment:
OK, I see in my reverse review that it is... though possibly only the once? As such, would suggest a more appropriate variable name (seems only used in relation to channel mapping)